cmd/memberagent/main.go

```diff
@@ -39,6 +39,7 @@ import (
 
 var (
 	scheme               = runtime.NewScheme()
+	useCaAuth            = flag.Bool("use-ca-auth", false, "Use identity and CA bundle to authenticate the member agent.")
```

Comment:
nit: Stylistically Go lang convention for acronyms is to match the casing used, e.g., `useCAAuth`.

Reply:
Thanks, changed ;-)